css/_variables.scss

```diff
@@ -89,7 +89,7 @@ $sidebarWidth: 200px;
  */
 $tooltipsZ: 901;
 $toolbarZ: 900;
-$overlayZ: 800;
```

Comment:
After https://github.com/jitsi/jitsi-meet/commit/98de4c90b543b95d5ba5e481e4180f6d32462528 the toolbar will be accessible for the ringing overlay. As for the others I can add a code that will hide the toolbars when the "message" type of overlay is shown, but not sure if we really need that.